src/cargo_ops/temp_project.rs

```diff
@@ -189,6 +189,7 @@ impl<'tmp> TempProject<'tmp> {
             to_update: Vec::new(),
             config: &self.config,
             dry_run: false,
+            workspace: false,
```

Comment:
This is the previous behaviour I believe. Perhaps worth exposing.

Reply:
We'll need to detect if this is a workspace, if so I think this resolves the old "can't find a version that matches". The MR is good to go with this though since there's no regression :)